src/GuiBase/Viewer/CameraInterface.cpp

```diff
@@ -36,6 +36,23 @@ Gui::CameraInterface::CameraInterface( uint width, uint height ) :
     setCameraZFar( 1000.0 );
 }
 
+void Gui::CameraInterface::resetToDefaultCamera() {
+    // get parameters from the current camera
+    // Thisis awfull and requires that the current camera is still alive ...
+    Scalar w = m_camera->getWidth();
+    Scalar h = m_camera->getHeight();
+    auto it = std::find_if(
+        Engine::SystemEntity::getInstance()->getComponents().cbegin(),
+        Engine::SystemEntity::getInstance()->getComponents().cend(),
+        []( const auto& c ) { return c->getName().compare( "CAMERA_DEFAULT" ) == 0; } );
+    if ( it != Engine::SystemEntity::getInstance()->getComponents().cend() )
+    {
+        m_camera = static_cast<Engine::Camera*>( ( *it ).get() );
+        m_camera->resize(w,h);
+        m_camera->show( false );
+    }

```

Comment:
Can we add a bit of logging here ? 
```suggestion
    }
    else 
    {
        LOG(logWARNING) << "A living camera is required. The application might now behave unexpectedly." ;
    }
```

Reply:
Why not, I applied your suggestion ...
But Remind that the else will never be reached except if the global invariant saying that a default camera is always created by the viewer when it is constructed (see the constructor above)

Reply:
Sure.
We should never reach that point, except, of course.... if we do ;)
I just wanted to avoid silent errors.